satellite/rewards/rewards.go

```diff
@@ -146,12 +171,55 @@ func (offers Offers) OrganizeOffersByType() OfferSet {
 			fc = append(fc, offer)
 		case Referral:
 			ro = append(ro, offer)
+		case Partner:
+			p = append(p, offer)
 		default:
 			continue
 		}
 	}
 
 	offerSet.FreeCredits = fc.OrganizeOffersByStatus()
 	offerSet.ReferralOffers = ro.OrganizeOffersByStatus()
+	offerSet.PartnerTables = organizePartnerData(p)
 	return offerSet
 }
+
+// createPartnerSet generates a PartnerSet from the config file.
+func createPartnerSet() PartnerSet {
+	partners := LoadPartnerInfos()
+	var ps PartnerSet
+	for _, partner := range partners {
+		ps = append(ps, OpenSourcePartner{
+			PartnerInfo: PartnerInfo{
```

Comment:
`partner` should already be of this type. Why copy its values over to another `PartnerInfo` instead of just doing `PartnerInfo: partner`?